Makefile

```diff
@@ -51,20 +51,20 @@ delimiter-%:
 	@echo '===================${GREEN} $* ${RESET}==================='
 
 fmt: deps ## Makes sure dep are installed and formats code
-	poetry run isort ff_fasttext_api
-	poetry run black ff_fasttext_api
+	poetry run isort category_api
+	poetry run black category_api
 
 lint: deps ## Makes sure dep are installed and lints code
 	poetry run ruff check .
 
 model: build-dev
-	docker-compose run -e RUST_BACKTRACE=1 --entrypoint poetry ff_fasttext_api run ffp-convert -f textdims ${INPUT_VEC} -t finalfusion ${OUTPUT_FIFU}
+	docker-compose run -e RUST_BACKTRACE=1 --entrypoint poetry category_api run ffp-convert -f textdims ${INPUT_VEC} -t finalfusion ${OUTPUT_FIFU}
 
 run: deps ## Runs category api locally using poetry uvicorn port: 28800
-	poetry run uvicorn ff_fasttext_api.main:app --host ${FF_FASTTEXT_API_CATEGORY_API_HOST} --port ${FF_FASTTEXT_API_CATEGORY_API_PORT}
+	poetry run uvicorn category_api.main:app --host ${CATEGORY_API_HOST} --port ${CATEGORY_API_PORT}
 
 run-container: deps build test_data/wiki.en.fifu ## Builds docker container, downloads fifu data and then runs docker container
```

Comment:
So as a general point on these make targets - the `run-container` is actually doing a build and run, which isn't really matching what's going on here. 

Additionally - due to the lack of source distribution, the `deps` step fails on a macOS machine which makes this further misleading. Either this needs correcting or very explicit callout to say this app cannot be run outside of Docker on a macOS machine.